### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by shriramshankar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,8 +51,9 @@ Refocus is a platform for visualizing the health and status of systems and/or se
 1. Install lunchy (`brew install Caskroom/cask/lunchy`). This will help you start redis.
 1. Run `lunchy start redis` to start redis.
 1. Run `npm start` to start your Node.js server at http://localhost:3000.
-1. If you want to unload some of the processing to a background process run `npm run start-clock`. If you intend to deploy this on heroku and have heroku toolbelt installed run `heroku local` to start both the web and the background process.
-
+1. To move the sample timeout check to a background clock process run, start the server by setting HEROKU_CLOCK_DYNO to true and then run  `npm run start-clock` to start the clock process.
```
Comment (iamigo) [inline README.md]: Does this only apply to heroku or could a separate process work a stand-alone non-heroku instance too?

```
1. If you're running in heroku and you want to run the sample timeout check in a separate background process:
    1. Start the server with environment variable (i.e. config setting) `HEROKU_CLOCK_DYNO` set to `true`.
    1. Run `npm run start-clock` to start the clock process.

File: README.md [hunk 2]
```diff
@@ -51,8 +51,9 @@ Refocus is a platform for visualizing the health and status of systems and/or se
 1. Install lunchy (`brew install Caskroom/cask/lunchy`). This will help you start redis.
 1. Run `lunchy start redis` to start redis.
 1. Run `npm start` to start your Node.js server at http://localhost:3000.
-1. If you want to unload some of the processing to a background process run `npm run start-clock`. If you intend to deploy this on heroku and have heroku toolbelt installed run `heroku local` to start both the web and the background process.
-
+1. To move the sample timeout check to a background clock process run, start the server by setting HEROKU_CLOCK_DYNO to true and then run  `npm run start-clock` to start the clock process. 
+1. To move the expensive bulkupsert to a web worker process, start the node server by setting the USE_WORKER_PROCESS to true and start the workerprocess by running `npm run start-worker`
```
Comment (iamigo) [inline README.md]: Does this only apply to heroku or could a separate process work a stand-alone non-heroku instance too?

```
1. If you're running in heroku and you want to offload expensive work from your web dyno(s) to separate worker dyno(s):
    1. Start the server with environment variable (i.e. config setting) `USE_WORKER_PROCESS ` set to `true`.
    1. Run `npm run start-worker` to start the worker process.

File: README.md [hunk 3]
```diff
@@ -51,8 +51,9 @@ Refocus is a platform for visualizing the health and status of systems and/or se
 1. Install lunchy (`brew install Caskroom/cask/lunchy`). This will help you start redis.
 1. Run `lunchy start redis` to start redis.
 1. Run `npm start` to start your Node.js server at http://localhost:3000.
-1. If you want to unload some of the processing to a background process run `npm run start-clock`. If you intend to deploy this on heroku and have heroku toolbelt installed run `heroku local` to start both the web and the background process.
-
+1. To move the sample timeout check to a background clock process run, start the server by setting HEROKU_CLOCK_DYNO to true and then run  `npm run start-clock` to start the clock process. 
+1. To move the expensive bulkupsert to a web worker process, start the node server by setting the USE_WORKER_PROCESS to true and start the workerprocess by running `npm run start-worker` 
+1. If you intend to deploy this on heroku and have heroku toolbelt installed run `heroku local` to start the web, the clock and the process together(Make sure that the HEROKU_CLOCK_DYNO and the USE_WORKER_PROCESS env variables are set to true).  The node server will be listning at port 5000, when you start it using the `heroku local` command.
```
Comment (iamigo) [inline README.md]:
```
+1. If you intend to deploy this on heroku and have heroku toolbelt installed, run `heroku local` to start the web, clock and worker processes all together. Make sure that the `HEROKU_CLOCK_DYNO` and `USE_WORKER_PROCESS` environment variables are set to true.  Note that the node server will be listening on port 5000 when you start it using the `heroku local` command.
```

**Comment by shriramshankar** [comment]
@iamigo PR updated.

**Comment by shriramshankar** [comment]
@iamigo merge conflicts fixed

**Comment by meynet-salesforce** [comment]
I added this to the quickstart so we can probably close this branch?

**Pull request closed without merging**